x/auth/README.md

```diff
@@ -20,18 +20,35 @@ This module is used in the Cosmos Hub.
 
 ## Contents
 
-* [Concepts](#concepts)
-    * [Gas & Fees](#gas--fees)
-* [State](#state)
-    * [Accounts](#accounts)
-* [AnteHandlers](#antehandlers)
-* [Keepers](#keepers)
-    * [Account Keeper](#account-keeper)
-* [Parameters](#parameters)
-* [Client](#client)
-  * [CLI](#cli)
-  * [gRPC](#grpc)
-  * [REST](#rest)
+- [`x/auth`](#xauth)
```

Comment:
Let's apply the correct `.md` formatting here too. (When you use the markdown all in one vscode extension, it unfortunately generates a ToC that does not respect the markdownlint config. You need to ctrl+z and ctrl+k+ctrl+s for reverting the formatting)

Reply:
Yea, Visual Studio's "Markdown all in one" doesn't play well with "MarkdownLint". I ended up just checking out all of my `.md` changes and updating it manually.